src/content_mapblock.cpp

```diff
@@ -472,10 +488,11 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 				for(u32 j=0; j<4; j++)
 				{
 					v3s16 neighbordir = cornerdir - halfdirs[j];
-					content_t content = neighbor_contents[neighbordir];
+
+					NeighbourData &neighbour_data = neighbour_datas[NeighbourToIndex(neighbordir)];
+					content_t& content = neighbour_data.content;
 					// If top is liquid, draw starting from top of node
-					if(neighbor_flags[neighbordir] &
-							neighborflag_top_is_same_liquid)
+					if (neighbour_data.flags & neighborflag_top_is_same_liquid)
```

Comment:
Sorry for nit picks, but two style errors:     `content_t &content` and { on same line as if